backend/src/main/java/gov/cdc/usds/simplereport/idp/authentication/LiveOktaAuthentication.java

```diff
@@ -190,7 +218,9 @@ public String enrollSmsMfa(String userId, String phoneNumber)
       return smsFactor.getId();
     } catch (ResourceException e) {
       if (e.getStatus() == HttpStatus.BAD_REQUEST.value()) {
-        throw new BadRequestException(e.getError().getMessage(), e);
+        throw new BadRequestException(
+            "Invalid phone number. You must enter a phone number capable of receiving text messages.",
```

Comment:
Long story short is that the message is always going to be invalid phone number (ditto for the exceptions below.) The longer story is worse:

There are effectively 2 kinds of errors we can generate here, one that the phone number is invalid and one that the user already has SMS MFA enrolled. Unfortunately for us, Okta doesn't seem to be doing a great job with error handling on their end, and only ever returns the message that the user is already enrolled in SMS, even if they aren't and the issue is that it's a landline 🙃 Since we don't really have a way to differentiate between those options, I kept this invalid phone number message since I think it's much more likely that users will try to enter a landline than be double-enrolled in SMS MFA.